package/Dockerfile

```diff
@@ -1,24 +1,29 @@
-FROM ghcr.io/oracle/oraclelinux:7-slim
-RUN yum -y update && \
-    yum -y install openssh && \
```

Comment:
openssh no longer needed?

Reply:
it is on the base image

Reply:
I checked the image rancher/gitjob:v0.3.0 and it was not on there